lstchain/datachecks/dl1_checker.py

```diff
@@ -154,10 +154,10 @@ def check_dl1(filenames, output_path, max_cores=4, create_pdf=False, batch=False
     # we assume that cam geom is the same in all files, & write the first one
     # we convert units from m to deg
     cam_description_table = \
-        Table.read(filenames[0], path='instrument/telescope/camera/LSTCam')
+        Table.read(filenames[0], path='configuration/instrument/telescope/camera/geometry_LSTCam')
```

Comment:
good opportunity to define that path as a global variable that we can import

Reply:
(and that can be used everywhere else)

Reply:
No, please just use `SubArrayDescription.to_hdf` and `SubArrayDescription.from_hdf`, no need for manually reading any tables.

Reply:
Indeed, even better

Reply:
Can a hardcoded name (`geometry_LSTCam`) be avoided?

Reply:
naming problems should go away when using `SubArrayDescription`

Reply:
changed